aspnetcore/mvc/views/partial.md

```diff
@@ -36,19 +36,17 @@ Partial views are created like any other view: you create a *.cshtml* file withi
 
 ## Referencing a Partial View
 
-From within a view page, there are several ways in which you can render a partial view. The simplest is to use `Html.Partial`, which returns an `IHtmlString` and can be referenced by prefixing the call with `@`:
-
-[!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=9)]
-
-The `PartialAsync` method is available for partial views containing asynchronous code (although code in views is generally discouraged):
+From within a view page, there are several ways in which you can render a partial view. The best practice is to use `Html.PartialAsync`, which returns an `IHtmlString` and can be referenced by prefixing the call with `@`:
 
 [!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=8)]
 
-You can render a partial view with `RenderPartial`. This method doesn't return a result; it streams the rendered output directly to the response. Because it doesn't return a result, it must be called within a Razor code block (you can also call `RenderPartialAsync` if necessary):
+You can render a partial view with `RenderPartialAsync`. This method doesn't return a result; it streams the rendered output directly to the response. Because it doesn't return a result, it must be called within a Razor code block:
+
+[!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=11-13)]
 
-[!code-cshtml[](partial/sample/src/PartialViewsSample/Views/Home/About.cshtml?range=10-12)]
+Because it streams the result directly, `RenderPartialAsync` may perform better in some scenarios. However, it's recommended you use `PartialAsync`.
 
-Because it streams the result directly, `RenderPartial` and `RenderPartialAsync` may perform better in some scenarios. However, in most cases it's recommended you use `Partial` and `PartialAsync`.
+While there are synchronous equivalents of `Html.PartialAsync` (`Html.Partial`) and `Html.RenderPartialAsync` (`Html.RenderPartial`), use of the synchronous equivalents isn't recommended because there are scenarios where they deadlock. The synchronous methods will be removed and unavailable in future versions.
```

Comment:
~~and unavailable~~

Reply:
The problem with that language is that they will only be removed *once*, so the sentence doesn't quite make sense imo. However, it could say ...

> The synchronous methods will be unavailable in future versions.

Reply:
You're right! Let's go with your suggested text instead.

Reply:
@scottsauber Go with ...

> The synchronous methods will be unavailable in future versions.